### PR TITLE
Readme - now with nice graph-ical improvements

### PR DESCRIPTION
**Pull request by sgarroum**
Added Explainer Graphs
Graphical improvements

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,15 +13,20 @@
 [![License](https://img.shields.io/github/license/airyhq/airy)](https://github.com/airyhq/airy/blob/develop/LICENSE)
 [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](https://github.com/airyhq/airy/projects)
 
-Airy Core is an open source, fully-featured, production ready messaging
+
+---
+
+![Airy_Explainer_Highlevel_Readme](https://user-images.githubusercontent.com/12533283/112460549-51e05d00-8d5f-11eb-802b-385c425dfaaf.png)
```
Comment (chrismatix) [inline README.md:19]: This a 1.3Mb png. Could you make it a jpeg and compress a little?

File: README.md [hunk 2]
```diff
@@ -60,42 +63,49 @@ Learn more about what we open-sourced in the
...
-### ⚡[APIs](https://airy.co/docs/core/api/introduction) to access your data
+
+- **⚡[APIs](https://airy.co/docs/core/api/introduction) to access your data
 
 An [API](https://airy.co/docs/core/api/introduction) to access conversational
 data with blazing fast HTTP endpoints.
 
-### 🔌[WebSockets](https://airy.co/docs/core/api/websocket) to power real-time applications
+
+- **🔌[WebSockets](https://airy.co/docs/core/api/websocket) to power real-time applications
```
Comment (lucapette) [inline README.md:90]:
```suggestion
- 🔌[WebSockets](https://airy.co/docs/core/api/websocket) to power real-time applications
```

File: README.md [hunk 3]
```diff
@@ -60,42 +63,49 @@ Learn more about what we open-sourced in the
...
 
 A [WebSocket server](https://airy.co/docs/core/api/websocket) that allows
 clients to receive near real-time updates about data flowing through the system.
 
-### 🎣[Webhook](https://airy.co/docs/core/api/webhook) to listen to events and participate programmatically in conversations
+
+- **🎣[Webhook](https://airy.co/docs/core/api/webhook) to listen to events and participate programmatically in conversations
```
Comment (lucapette) [inline README.md:96]:
```suggestion
- 🎣[Webhook](https://airy.co/docs/core/api/webhook) to listen to events and participate programmatically in conversations
```

File: README.md [hunk 4]
```diff
@@ -60,42 +63,49 @@ Learn more about what we open-sourced in the
...
 
 A webhook integration server that allows its users to programmatically
 participate in conversations by sending messages (the webhook integration
 exposes events users can "listen" to and react programmatically.)
 
-### 💎[UI: From an inbox to dashboards](https://airy.co/docs/core/apps/ui/introduction)
+
+- **💎[UI: From an inbox to dashboards](https://airy.co/docs/core/apps/ui/introduction)
```
Comment (lucapette) [inline README.md:103]:
```suggestion
- 💎[UI: From an inbox to dashboards](https://airy.co/docs/core/apps/ui/introduction)
```

File: README.md [hunk 5]
```diff
@@ -60,42 +63,49 @@ Learn more about what we open-sourced in the
...
+plugin](https://airy.co/docs/core/sources/chat-plugin) to Facebook
+Messenger & Google's Business Messages to your Airy Core. This is
 all possible through an ingestion platform that heavily relies on [Apache
 Kafka](https://kafka.apache.org) to process incoming webhook data from different
 sources. We make sense of the data and reshape it into source independent
 contacts, conversations, and messages.
 
-### ⚡[APIs](https://airy.co/docs/core/api/introduction) to access your data
+
+- **⚡[APIs](https://airy.co/docs/core/api/introduction) to access your data
```
Comment (lucapette) [inline README.md:84]:
```suggestion
- ⚡[APIs](https://airy.co/docs/core/api/introduction) to access your data
```

File: README.md [hunk 6]
```diff
@@ -60,42 +63,49 @@ Learn more about what we open-sourced in the
...
 
+---
 ## Components
 
+![Airy_Explainer_Components_Readme (1)](https://user-images.githubusercontent.com/12533283/112460661-6de3fe80-8d5f-11eb-8274-8446fbfcf5c8.png)
+
 Airy Core contains the following components:
 
-### 💬 Connectors for all [conversational sources](https://airy.co/docs/core/sources/introduction)
+- **💬 Connectors for all [conversational sources](https://airy.co/docs/core/sources/introduction)
```
Comment (lucapette) [inline README.md:73]:
```suggestion
- 💬 Connectors for all [conversational sources](https://airy.co/docs/core/sources/introduction)
```

**Review by lucapette — CHANGES_REQUESTED** [review]
Applying the suggestions will fix the formatting problem.

I do agree with @chrismatix we should compress the image a little

File: README.md (hunk 1 above)
Comment (lucapette) [inline README.md:19]:
```suggestion
![Airy_Explainer_Highlevel_Readme](https://user-images.githubusercontent.com/124274/113720584-18a8d500-96ef-11eb-97c3-362eebd6253d.jpeg)
```

**Review by lucapette — APPROVED** [review]

**Pull request merged**